docs/cli.rst

```diff
@@ -1,11 +1,248 @@
+Once you know the specifc task or flow you want to run, they can be executed with the ``run`` command:
+
+.. code-block:: console
+
+    $ cci task run task_name
+    $ cci flow run flow_name
+
+Where ``task_name`` and ``flow_name`` are the actual name of the task or flow that you would like to run.
+Tasks usually require additional options to be passed when using the ``cci task run`` command.
+See the next section for how to view task specific option information. 
+
+
+Getting More Information
+*******************************
+For additional information on tasks use::
+
+    $ cci task info <task_name>
+
+Where ``<task_name>`` is the name of a specific task.
+Information about specific tasks includes:
+
+* A description of what the task does
+* The specific Python class associated with this task
+* The syntax for running the command
+* Any options for the task
+
+Information on specific options includes:
+
+* The syntax for the option (``-o option_name value``).
+* If the options is required or optional.
+* A description of each option.
+
+Example output looks like this:
+
+.. code-block:: console
+
+    $ cci task info util_sleep
+    util_sleep
+
+    Description: Sleeps for N seconds
+
+    Class: cumulusci.tasks.util.Sleep
+
+    Command Syntax
+
+        $ cci task run util_sleep
+
+    Options
+
+        -o seconds SECONDS
+        Required
+        The number of seconds to sleep
+        Default: 5
+
+For additional information on flows use::
+
+    $ cci flow info <flow_name>
+
+Where ``<flow_name>`` is the actual name of the flow.
+Information on specific flows includes:
+
+* A description of the flow.
+* The ordered steps (and substeps) of a flow.
+
+Example output looks like this:
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Troubleshooting Errors
+----------------------
+
+Errors happen! That’s why our team strives to provide our users with options for efficiently working with them when they occur.
```

Comment:
Should we include some information about where to come for help (Trailblazer Community, Power of Us Hub) with errors and stack traces?

Reply:
@davidmreed - Yes, I think this is something that definitely belongs here and is missing currently. I'm not sure if we want to also include the topic of opening a GitHub issue in this part as well. Another potential home for that could be under the "Contribute" section.

Reply:
I don't think "Contribute" is an obvious place to look if you're seeking help. Here is probably better.